### PR TITLE
clickselect improvements

### PR DESCRIPTION
**Pull request by atomiomi**

### CODE REVIEW

File: src/click_select/index.es [hunk 1]
```diff
@@ -10,7 +10,28 @@ class ClickSelect extends PureComponent {
     this.handleClick = this.handleClick.bind(this);
   }
 
-  handleClick(event) {
+  copyToClipboard() {
+    if (!document.queryCommandSupported) return false;
```
Comment (suprMax) [inline src/click_select/index.es]: read browser specifics here: https://stackoverflow.com/questions/400212/how-do-i-copy-to-the-clipboard-in-javascript

try to use the new Clipboard API first.

File: src/click_select/index.es [hunk 2]
```diff
@@ -10,7 +10,28 @@ class ClickSelect extends PureComponent {
...
+    this.select();
+    let result;
+
+    try {
+      result = document.execCommand('copy');
+    } catch (err) {
+      result = false;
+    }
+
+    this.unselect();
```
Comment (suprMax) [inline src/click_select/index.es]: no need to deselect

File: src/click_select/index.es [hunk 3]
```diff
@@ -10,7 +10,28 @@ class ClickSelect extends PureComponent {
...
+    return result;
```
Comment (suprMax) [inline src/click_select/index.es]: do you need to return result?

Comment (atomiomi) [reply src/click_select/index.es]: yep, need to show in result if a copy to the clipboard was successed

**Pull request merged**